src/manifest.c

```diff
@@ -848,7 +848,7 @@ struct list *filter_out_existing_files(struct list *files)
 		file = list->data;
 
 		filename = mk_full_filename(path_prefix, file->filename);
-		if (verify_file_lazy(filename)) {
+		if (verify_file(file, filename)) {
```

Comment:
verify_file() is very slow and calling it here would slow down the bundle-add significantly.
And using verify_file_lazy() is terrible too, because it doesn't make sense. We need to check if the file that is being added isn't present in a current Manifest and not in the system.

Reply:
I don't follow, the file will obviously be in a manifest, that is why it is being added to the system with bundle-add. Or do you mean in the manifest of all other bundles already installed in the system?

Reply:
When we are installing files from a BundleA instead of checking if those files are present and correct in the system we need to check if they are present in any manifest of a bundle that is assumed as installed.

There's a big overlap between files in bundle Manifests. Checking all files that are installed by a bundle is to expensive.